xml/hardware_nvdimm.xml

```diff
@@ -304,7 +296,8 @@
   <sect2>
    <title>PMEM with BTT</title>
    <para>
-    This is most useful when you just want to use NVDIMMs as very fast storage.
+    This is most useful when you just want to use the persistent memory on a set
```

Comment:
remove 'just' (style guide)

Reply:
Fair point! I have simplified this sentence slightly.